packages/validators-util/src/isValidRegistreringsnummer/isValidRegistreringsnummer.ts

```diff
@@ -0,0 +1,3 @@
+export const REGNR_REGEX = /^[a-zA-Z]{2}\d{5}/;
+
+export const isValidRegistreringsnummer = (value: string) => REGNR_REGEX.test(value);
```

Comment:
Nå jobber ikke jeg med bil, men er ikke denne for lite spesifik? Eller er tanket at Registeringsnummer er default for vanlig bil. Også får man mer spesifike for feks RegisteringsnummerMC, veteran osv?

Reply:
Dette er bare for bil ja, burde ha spesifisert det. Er det best å ha alt i en regex eller lage spesifikke regler for hvert kjøretøy?